### PR TITLE
Update cv depedency

### PR DESCRIPTION
**Pull request by AdamPalmarUnity**
# Peer Review Information

Add tests for create_estimator

# Pull Request Check List

- [ ] Added **tests** for changed code.
- [ ] Updated **documentation** for changed code.

### CODE REVIEW

File: tests/test_create_estimator.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import pytest
+from yacs.config import CfgNode as CN
+
+from datasetinsights.estimators import FasterRCNN, create_estimator
+
+
+@pytest.fixture
+def config():
```
Comment (adason) [inline tests/test_create_estimator.py:8]: This test only tries to create estimator for `FasterRCNN`, as this is the only estimator defined in the yaml.

File: tests/test_create_estimator.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    """prepare config."""
+    with open("tests/configs/faster_rcnn_groceries_real_test.yaml") as f:
+        cfg = CN.load_cfg(f)
+
+    return cfg
+
+
+def test_create_estimator_with_config_expect_faster_rcnn(config):
```
Comment (adason) [inline tests/test_create_estimator.py:16]: Test against a specific estimator create will become tedious,  as developers have to remember add a new test for each new estimator that was introduced. It's very likely that developers will forgot. Test whether `FasterRCNN` estimator can be successfully initialized should be the responsibility of tests specific to each estimator, such as `test_faster_rcnn.py` or `test_deeplab.py`

File: tests/test_create_estimator.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+    estimator = create_estimator(name=config.estimator, config=config,)
```
Comment (adason) [inline tests/test_create_estimator.py:18]: We should test if `_find_estimator` is working properly instead testing if a particular estimator can be initialized.

**Comment by adason** [comment]
The title of this PR states `Update cv depedency` but it only modified python code without changing dependencies in either `pyproject.toml` or `Dockerfile`. Are those changes missing in this PR?

**Comment by AdamPalmarUnity** [comment]
Yes the update to the depedency is not there yet.

**Pull request closed without merging**